app/src/components/forms/DepositBankForm.vue

```diff
@@ -19,10 +20,29 @@
 import LoadingButton from '@/components/LoadingButton.vue'
 import { NETWORK } from '@/constant'
 import { ref } from 'vue'
+import { required, numeric } from '@vuelidate/validators'
+import { useVuelidate } from '@vuelidate/core'
 
 const amount = ref<string>('0')
-defineEmits(['deposit', 'closeModal'])
+const emits = defineEmits(['deposit', 'closeModal'])
 defineProps<{
   loading: boolean
 }>()
+
```

Comment:
can you add not zero validation like this
```typescript
const notZero = helpers.withMessage('Amount must be greater than 0', (value: string) => {
  return parseFloat(value) > 0
})
```

Reply:
Done